sickbeard/ui.py

```diff
@@ -39,7 +41,13 @@ def message(self, title, message=''):
         title: The title of the notification
         message: The message portion of the notification
         """
-        self._messages.append(Notification(title, message, MESSAGE))
+        #self._messages.append(Notification(title, message, MESSAGE))
+        new_notification = Notification(title, message, MESSAGE)
+
+        push_to_web_socket(json.dumps({'event': 'notification',
```

Comment:
Here it is sending the json. Previously it was queueing up the notifications, and keeping track of the clients with there source ip, to make sure if there were two clients, client one did not remove the notification from the queue, and as a result making sure client two could not get the notification anymore.

I believe by echoing to all clients, we don't need this logic anymore?!